packages/sit-onyx/src/components/OnyxVisuallyHidden/OnyxVisuallyHidden.ct.tsx

```diff
@@ -0,0 +1,20 @@
+import { expect, test } from "../../playwright/a11y";
+import OnyxVisuallyHidden from "./OnyxVisuallyHidden.vue";
+
+test("should not be visible", async ({ mount }) => {
+  const component = await mount(
+    <div>
```

Comment:
```suggestion
    <div style={{ width: "max-content" }}>
```

So the screenshots are smaller and easier to read